database/DataGenerator/createHouseData.js

```diff
@@ -0,0 +1,247 @@
+const fs = require('fs');
+const faker = require('faker');
+const amenityList = require('./amenityList');
+const bedSize = require('./bedSize');
+
+const numOfRecords = 10000000;
+
+const generateHouseData = (filepath, length) => {
+  return new Promise((resolve, reject) => {
+    const writeStream = fs.createWriteStream(filepath, { encoding: 'utf8' });
+    writeStream.write('house_id,title,location,is_entire_place,super_host_name,super_host_photo,rating,desc,space_desc\n');
+
+    (async() => {
+      for (let i = 1; i <= length; i++) {
+        const house_id = i;
+        const title = faker.lorem.sentences(2);
+        const location = faker.address.city();
+        const is_entire_place = faker.random.boolean();
+        const super_host_name = `${faker.name.firstName()} ${faker.name.lastName()}`;
+        const super_host_photo = 'super-host-photo-1.jpg';
+        const rating = faker.random.number(100);
+        const desc = faker.lorem.sentences(3);
+        const space_desc = faker.lorem.sentences(10);
+
+        if(!writeStream.write(`${house_id},${title},${location},${is_entire_place},${super_host_name},${super_host_photo},${rating},${desc},${space_desc}`)) {
+          await new Promise(resolve => writeStream.once('drain', resolve));
+        }
+        if (i !== length) {
+          writeStream.write('\n');
+        }
+      }
+      writeStream.end();
+    })();
+    writeStream.on('finish', () => resolve('house data generated'));
+  });
+};
+
+const generatePhotoData = (filepath, length) => {
+  return new Promise((resolve, reject) => {
+
+    const writeStream = fs.createWriteStream(filepath, { encoding: 'utf8' });
+    writeStream.write('house_id,file_path,desc\n');
+
+    (async() => {
+      for (let i = 1; i <= length; i++) {
+        const numOfPictures = faker.random.number({min: 1, max:3});
+        for (let j = 0; j <= numOfPictures; j++) {
+          const house_id = i;
+          const file_path = `HousePic${faker.random.number({min: 1, max: 910})}.jpg`;
+          const desc = faker.lorem.sentences(2);
+
+          if(!writeStream.write(`${house_id},${file_path},${desc}`)) {
+            await new Promise(resolve => writeStream.once('drain', resolve));
+          }
+          if (i !== length || j !== numOfPictures) {
+            writeStream.write('\n');
+          }
+        }
+      }
+      writeStream.end();
+    })();
+    writeStream.on('finish', () => resolve('picture data generated'));
+  });
+};
+
```

Comment:
@AndrewTjandra I might suggest adding a comment so you have it for reference later conceptually distinguishing between the amenity/amenities list functions without reading the code.